.clang-tidy

```diff
@@ -82,6 +83,7 @@ Checks: >
     -readability-make-member-function-const,
     -readability-redundant-string-init,
     -readability-non-const-parameter,
+    -readability-container-contains,
```

Comment:
How about adding a ticket into the backlog to fix these warnings?

Reply:
Sure, https://github.com/Project-OSRM/osrm-backend/issues/6902